MAUI/Kanban-Board/Cards.md

```diff
@@ -213,4 +213,6 @@ kanban.CardTemplate = new KanbanTemplateSelector();
       
 {% endhighlight %}
 
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+N> When using a custom model as the [`ItemsSource`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Kanban.SfKanban.html#Syncfusion_Maui_Kanban_SfKanban_ItemsSource), the binding context for each card is set to an instance of that custom model. This means that all data bindings within the [`CardTemplate`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Kanban.SfKanban.html#Syncfusion_Maui_Kanban_SfKanban_CardTemplate) should reference the properties of your custom model directly.
```

Comment:
Use below content 

- When a custom data model is assigned to the `ItemsSource` property of the `SfKanban` control, each card's `BindingContext` is automatically set to an instance of that model. Therefore, bindings within the `CardTemplate` must directly reference the properties defined in the custom model.

- The default card UI is not applicable when using a custom model. To render the card content correctly, you must define a custom `DataTemplate` using the `CardTemplate` property.

**Move these changes to here** 

<img width="532" height="151" alt="image" src="https://github.com/user-attachments/assets/f58982fc-6072-4412-b903-8940e88ac94d" />